vars/daosLatestVersion.groovy

```diff
@@ -28,6 +28,32 @@ String distro2repo(String distro) {
     }
 }
 
+String getLatestVersion(String type='stable')
+{
+    String v = null
+    try {
```

Comment:
Let's use a variable here, you will see why below.

```suggestion
    try {
    repo = 'daos-stack-daos-' + distro2repo(_distro) + '-x86_64-' + type + '-local/'
```

Reply:
Done